apps/backend/package.json

```diff
@@ -3,16 +3,16 @@
     "version": "0.0.1",
     "description": "Backend for AntAlmanac",
     "scripts": {
-        "dev": "esbuild --watch --bundle --sourcemap --outdir=dist --platform=node src/index.ts",
-        "build": "esbuild --bundle --minify --outdir=dist --platform=node src/lambda.ts",
-        "start": "concurrently 'npm run dev' 'nodemon dist'",
+        "dev": "tsx src/index.ts",
```

Comment:
Does `tsx` also live-reload the program when the source changes? I think that was the goal of the convoluted dev command before.

Reply:
Oh, yeah it can

`tsx watch src/index.ts`